python/res/fm/ecl/ecl_run.py

```diff
@@ -338,3 +346,18 @@ def checkCase(cls , refcase , simcase):
 Simulation %s : %s
 """ % ( refcase , ref.getEndTime() , simcase , sim.getEndTime())
             raise ValueError( msg )
+
+    @staticmethod
+    def _await_process(process, out_file):
```

Comment:
How about `_await_process_tee`? That would describe the function's main features:
- Block (wait) until process finishes.
- Tee output into stdout and file.

I agree that consumers of the function might expect it to take a list of files, since that's what the `tee` program does. However, at the moment we don't need this functionality, so it's probably unwise to overcomplicate the function.

Reply:
Agreed!